### DISCUSSION TITLE
webdav CORS

### DISCUSSION THREAD

**Discussion by AmbroseX** [remotely-save/remotely-save#14] [Q&A] [answered]
Many webdav do not support CORS, Please let the plugin support the webdav protocol without CORS

**Comment by exoticknight** [comment]
CORS is controlled by browser, I thnik it's not much things that plugins can deal with it 🥲

**Reply by AmbroseX** [reply]
But other app can sync by webdav without cors like Zotero

**Reply by exoticknight** [reply]
like I said, CORS is controlled by browser, and mobile version of obsidian use browser or webkit engine (heart of browser) to render plugin's UI.
Plugins syncing with webdev is much like sending request in browser, without the help of obsidian, CORS is a invincible wathdog.
Other apps works fine, simply because they are at the same layer like obsidian, which can send request by itself, not in browser.

**Comment by fyears** [comment, marked as answer]
It depends on Obsidian [exposing a more-featured native method](https://github.com/obsidianmd/obsidian-api/issues/35) to bypass cors. Before that I am afraid there’s not so much I could do.

**Reply by AmbroseX** [reply]
like Zotero can sync by webdav without cors

**Reply by chuanwenzhao** [reply]
请问支持坚果云吗

**Reply by Satisfy8375** [reply]
不支持自建webdav啊
